script/20240516-revoke-roles/02_GrantAdminRoleForMultisig_Mainnet.s.sol

```diff
@@ -0,0 +1,362 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import { console } from "forge-std/console.sol";
+import { Migration } from "script/Migration.s.sol";
+import { Contract } from "script/utils/Contract.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { ERC721 } from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+import { AccessControlEnumerable } from "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
+import { LibRNSDomain } from "src/libraries/LibRNSDomain.sol";
+import { DefaultNetwork } from "@fdk/utils/DefaultNetwork.sol";
+import { RNSUnified } from "src/RNSUnified.sol";
+import { RNSAuction } from "src/RNSAuction.sol";
+import { NameChecker } from "src/NameChecker.sol";
+import { RNSDomainPrice } from "src/RNSDomainPrice.sol";
+import { PublicResolver } from "src/resolvers/PublicResolver.sol";
+import { RNSReverseRegistrar } from "src/RNSReverseRegistrar.sol";
+import { RONRegistrarController } from "src/RONRegistrarController.sol";
+import { OwnedMulticaller } from "src/utils/OwnedMulticaller.sol";
+import { INSDomainPrice } from "src/interfaces/INSDomainPrice.sol";
+import { OwnedMulticallerDeploy } from "script/contracts/OwnedMulticallerDeploy.s.sol";
+import { ErrorHandler } from "src/libraries/ErrorHandler.sol";
+import { EventRange } from "src/libraries/LibEventRange.sol";
+import { RNSOperation } from "src/utils/RNSOperation.sol";
+import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
+import { INSAuction } from "src/interfaces/INSAuction.sol";
+import { INSDomainPrice } from "src/interfaces/INSDomainPrice.sol";
+
+contract Migration__02_GrantAdminRoleForMultisig_Mainnet is Migration {
+  using Strings for *;
+  using ErrorHandler for bool;
+  using LibRNSDomain for string;
+
+  address duke = 0x0F68eDBE14C8f68481771016d7E2871d6a35DE11;
+  address multisig = 0x1FF1edE0242317b8C4229fC59E64DD93952019ef;
```

Comment:
should it be `0x9d05d1f5b0424f8fde534bc196ffb6dd211d902a`?
```suggestion
  address multisig = 0x9d05d1f5b0424f8fde534bc196ffb6dd211d902a;
```

Reply:
Correct